### ISSUE TITLE
<MrWeak> nixtux: can you please remove the feep-feep-feep sound from arm snipers? it is a bit annoying

### ISSUE THREAD

**Issue by nixtux** [n3wm1nd/TA#975]

**Comment by Gambit99** [comment]
I can look into this. I hate it too.

**Comment by nixtux** [comment]
Has it only started to happen in latest release?

**Comment by Gambit99** [comment]
I don't know but arm snipers are entirely bugged. I loaded a sandbox and gave 20 snipers and 20 construction kbots and the snipers can't hit the cons a large portion of the time. And, their laser beam sight seems to do some kind of minor damage instead. It's weird.

**Comment by Zonkspring** [comment]
Its been there for a while now I think - at least a Month.
Imo the laser beam is probably meant to do a bit of dmg.
Also, I believe it is that laser beam thats making the annoying feep-feep-feep sound.

**Comment by Gambit99** [comment]
I removed the laser sight beam damage -- it takes minutes for that effect to kill even a simple constructor, and it is firing 10x per second per sniper! That will cause a lot of unecessary projectile checks and cpu lag, so I just removed it. This is in #976

**Comment by skymyj** [comment]
It's normal that hurt a little. Are you sure youd had enough energy? :)

**Comment by nixtux** [comment]
The laser is designed to deal no damage, if it is that's a serious bug, this will also effect all units with a sniper weapon just just armsnipe, it may be easier if I just revert original commit, will check later

**Comment by Gambit99** [comment]
No -- I saw a few of the constructors at 99% .. 98% .. 97% health. If there are only snipers and constructors in the sandbox then the constructors should never reach these numbers. I gave myself an afus so that I'd have enough energy. I see that the damage is 0 in the definition but the only thing hitting them is the snipers and they are clearly taking damage. Is it possible that they take this damage from the "kick" of the gun? Or that there is a random component to the damage?

In any case, reloadtime=0.1 at long range means lots of long line projectile checks. I'm sure an i7 would handle it but it is still a rather expensive effect. There are only a few units that have this like beamers (relatively short range) and the furie (relatively fewer in number). Mines and the super krog also both have a 0.1, but those weapons are more of a short range proximity check.
